articles/introduction-to-extended-reality/index.md

```diff
@@ -0,0 +1,73 @@
+# Introduction to Extended Reality (XR)
+
+## What is Extended Reality (XR)?
+Extended reality (XR) is a term that encompasses real and virtual environments that are generated by wearable devices or computer technology to provide an immersive experience. It can also be described as a collection of all immersive technologies that combine real and virtual worlds. XR is growing in terms of advancement and application. This technology is still evolving because of ongoing development in human-machine interactions. 
+
+## Pillars of Extended Reality
+Extended reality encompasses three main pillars of immersive technology: Virtual Reality (VR), Augmented Reality (AR) and Mixed Reality (MR). 
+### Virtual Reality (VR)
+This immersive technology employs a head mounted display (HMD) to provide an immersive expression through simulation. The simulated digital environment makes users feel as if they are there. For example, VR developers may create an artificial world that makes users feel as if they are walking around on the moon. The first industries to adopt this kind of technology include the gaming and entertainment industries.
+
+![Virtual Reality](/engineering-education/introduction-to-extended-reality/virtual-reality.jpg)
+
+[Image Source: Magic Holo](https://magic-holo.com/wp-content/uploads/2017/04/Virtual_Reality.jpg)
+
+### Augmented Reality (AR)
+In this component of XR, smart devices are used to overlay sounds and digital graphics into the real world. Some of the digital details overlaid into the real-world environment include images, animation and texts. This enhanced experience can be achieved using AR screens, tablets, smartphones or screens. Snapchat filters and Pokeman Go are good examples of AR technology. In the game of Pokemon Go, digital creatures are overlaid into the real world. Snapchat filters involve putting digital objects such as glasses and hats onto the user’s head to provide unique experiences.
+
+![Augmented Reality](/engineering-education/introduction-to-extended-reality/augmented-reality.jpeg)
+
+[Image Source: Miro](https://miro.medium.com/max/1200/1*3F8SgP5Im17Kq9yPvy5rpA.jpeg)
+
+### Mixed Reality (MR)
+Mixed Reality encompasses the co-existence of real-world and digital objects. There is an interaction of these objects in real time. This is an advanced immersive technology that needs an MR headset and higher processing power than virtual reality and augmented reality. A good example of mixed reality is Microsoft's HoloLens, which allows users to interact with digital objects that are placed in a room.
+
+## How Extended Reality Works

```

Comment:
Also, can you mention how AR, VR, and MR work to enable XR? What are their respective roles?

Reply:
I have added a paragraph to incorporate this suggestion.